encord/http/error_utils.py

```diff
@@ -15,92 +15,13 @@
 
 from encord.exceptions import *
 
-# Error messages
-AUTHENTICATION_ERROR = ["AUTHENTICATION_ERROR"]
-AUTHORISATION_ERROR = ["AUTHORISATION_ERROR"]
-RESOURCE_NOT_FOUND_ERROR = ["RESOURCE_NOT_FOUND_ERROR"]
-METHOD_NOT_ALLOWED_ERROR = ["METHOD_NOT_ALLOWED_ERROR"]
-UNKNOWN_ERROR = ["UNKNOWN_ERROR"]
-OPERATION_NOT_ALLOWED_ERROR = ["OPERATION_NOT_ALLOWED"]
-ANSWER_DICTIONARY_ERROR = ["ANSWER_DICTIONARY_ERROR"]
-CORRUPTED_LABEL_ERROR = ["CORRUPTED_LABEL_ERROR"]
-FILE_TYPE_NOT_SUPPORTED_ERROR = ["FILE_TYPE_NOT_SUPPORTED_ERROR"]
-FILE_SIZE_NOT_SUPPORTED_ERROR = ["FILE_SIZE_NOT_SUPPORTED_ERROR"]
-FEATURE_DOES_NOT_EXIST_ERROR = ["FEATURE_DOES_NOT_EXIST_ERROR"]
-MODEL_WEIGHTS_INCONSISTENT_ERROR = ["MODEL_WEIGHTS_INCONSISTENT_ERROR"]
-MODEL_FEATURES_INCONSISTENT_ERROR = ["MODEL_FEATURES_INCONSISTENT_ERROR"]
-UPLOAD_OPERATION_NOT_SUPPORTED_ERROR = ["UPLOAD_OPERATION_NOT_SUPPORTED_ERROR"]
-MUST_SET_DETECTION_RANGE_ERROR = ["MUST_SET_DETECTION_RANGE_ERROR"]
-DETECTION_RANGE_INVALID_ERROR = ["DETECTION_RANGE_INVALID_ERROR"]
-RESOURCE_EXISTS_ERROR = ["RESOURCE_EXISTS_ERROR"]
-INVALID_DATE_FORMAT_ERROR = ["INVALID_DATE_FORMAT_ERROR"]
-
 
 def check_error_response(response, payload=None):
```

Comment:
As stated, I disagree with the general approach here. But if we *are* taking it, then we should get rid of the `check_error_response` function altogether and just raise the exception directly. The func here is doing the "dynamic dispatch", and if we get rid of it then it looses its purpose.